docs/guide/twig_functions_reference.md

```diff
@@ -153,7 +153,7 @@ If the Content item does not have a translation in the current language, the mai
 
 #### Description
 
-`ez_field()` is a Twig helper which returns a Field value in the current language.
+`ez_field()` is a Twig helper which returns a Field  in the current language. The field gives you access to the field value, as well as the Field's Definition and Type identifiers.
```

Comment:
`Field's Definition` Maybe make it more clear we are referring Field Definition identifier, as to not give impression we give field Definition.

aka `Field's Definition identifier and Type identifier`

Reply:
I'd say the current version is fine, but people *could* misunderstand it, and adding the second `identifier` will prevent it.